types/es6-shim/index.d.ts

```diff
@@ -556,7 +555,7 @@ interface Map<K, V> {
     clear(): void;
     delete(key: K): boolean;
     forEach(callbackfn: (value: V, index: K, map: Map<K, V>) => void, thisArg?: any): void;
-    get(key: K): V;
+    get(key: K): V | undefined;
     has(key: K): boolean;
     set(key: K, value?: V): Map<K, V>;
```

Comment:
That would be exactly like the current ES6 definition in TypeScript https://github.com/Microsoft/TypeScript/blob/master/lib/lib.es6.d.ts#L4826